docs/madr/decisions/080-inbound-policies-matches.md

```diff
@@ -207,40 +648,6 @@ policies:
         - kri: kri_mtp_...
 ```
 
-##### Extensibility with DPP labels selector
```

Comment:
should we have mention why we're removing this?

Reply:
explained in https://github.com/kumahq/kuma/pull/13848/files#diff-45dce14e9572bcf179427f441dbe76eba3742cf03befd54964e96b41e37847e0R468